core/src/main/scala/org/apache/spark/Dependency.scala

```diff
@@ -59,6 +60,9 @@ abstract class NarrowDependency[T](_rdd: RDD[T]) extends Dependency[T] {
   override def rdd: RDD[T] = _rdd
 }
 
+object ShuffleDependency {
+  private val EMPTY_ROW_BASED_CHECKSUMS: Array[RowBasedChecksum] = Array.empty
```

Comment:
nit: We can make this `private[spark]` and use it in other places within this PR